tests/shared/core/training_data/test_visualization.py

```diff
@@ -188,3 +188,21 @@ def test_story_visualization_with_merging(domain: Domain):
     assert 15 < len(generated_graph.nodes()) < 33
 
     assert 20 < len(generated_graph.edges()) < 33
+
+
+def test_remove_auxiliary_nodes():
+    # Create a sample graph
+    graph = nx.MultiDiGraph()
```

Comment:
`nx` is not defined just add `from networkx import MultiDiGraph`

Reply:
@Urkem, Done!